lib/nimble_template/addons/credo.ex

```diff
@@ -23,9 +23,11 @@ defmodule NimbleTemplate.Addons.Credo do
   end
 
   defp inject_mix_dependency(project) do
-    Generator.inject_mix_dependency(
-      {:credo, latest_package_version(:credo), only: [:dev, :test], runtime: false}
-    )
+    Generator.inject_mix_dependency([
+      {:credo, latest_package_version(:credo), only: [:dev, :test], runtime: false},
+      {:compass_credo_plugin, latest_package_version(:compass_credo_plugin),
```

Comment:
Should we add this into the `credo` test @hanam1ni

Reply:
Added test in [297a251](https://github.com/nimblehq/elixir-templates/pull/276/commits/297a251f9e02e3b223b4e799f2c0a9bca52294b5)